self-core-impl/src/test/java/com/selfxdsd/core/mock/InMemoryProjectsTestCase.java

```diff
@@ -81,6 +83,37 @@ public void projectOwnedBy() {
                 Matchers.iterableWithSize(1));
     }
 
+    /**
+     * Check if the right project is returned.
+     */
+    @Test
+    public void getProjectById() {
+        final Storage storage = new InMemory();
+        ProjectManager projectManager = storage
+                .projectManagers().pick("github");
+        InMemoryProjects projects = (InMemoryProjects)
```

Comment:
@BogdanAdrian11 you don't need any cast. ``Projects all = storage.projects()`` is enough.